### PR TITLE
Feat: add documentation to test a secure route with a JWT token in Swagger

### PR DESCRIPTION
**Pull request by toofff**

### CODE REVIEW

File: core/jwt.md [hunk 1]
```diff
@@ -59,6 +59,40 @@ security:
         - { path: ^/, roles: [ ROLE_READER ] }
 ```       
 
+## Testing with Swagger
+
+You want to test the routes of your API protected by JWT authentication?
+
+### Configure api platform
```
Comment (dunglas) [inline core/jwt.md]: `Configuring API Platform`

File: core/jwt.md [hunk 2]
```diff
@@ -59,6 +59,40 @@ security:
...
+
+```yml
+# app/config/config.yml
+
+api_platform:
+    swagger:
+         api_keys:
+            - { name: 'Authorization', type: 'header' }
```
Comment (dunglas) [inline core/jwt.md]: 1 space missing

File: core/jwt.md [hunk 3]
```diff
@@ -59,6 +59,40 @@ security:
...
+```
+
+And the Authorise button will appear.
```
Comment (dunglas) [inline core/jwt.md]: `Authorize`

File: core/jwt.md [hunk 4]
```diff
@@ -59,6 +59,40 @@ security:
...
+
+![Screenshot of API Platform with Authorize button](images/JWTAuthorizeButton.png)
+
+### Add new API key
```
Comment (dunglas) [inline core/jwt.md]: `Adding a New API Key`

File: core/jwt.md [hunk 5]
```diff
@@ -59,6 +59,40 @@ security:
...
+
+All you have to do is configure the API key in the value field.
```
Comment (dunglas) [inline core/jwt.md]: `to configure`

File: core/jwt.md [hunk 6]
```diff
@@ -59,6 +59,40 @@ security:
...
+
+By default [only the authorization header mode is enabled](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#2-use-the-token) in [LexikJWTAuthenticationBundle](https://github.com/lexik/LexikJWTAuthenticationBundle).
+
+You must add the [JWT token](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#1-obtain-the-token) as below and authorize.
```
Comment (dunglas) [inline core/jwt.md]: `set` instead of `add`? `and click on the "Authorize" button`?

**Comment by toofff** [comment]
@dunglas done ;)

File: core/jwt.md [hunk 7]
```diff
@@ -59,6 +59,40 @@ security:
...
+             - { name: 'Authorization', type: 'header' }
+```
+
+And the Authorize button will appear.
+
+![Screenshot of API Platform with Authorize button](images/JWTAuthorizeButton.png)
+
+### Adding a New API Key
+
+All you have to do is to configure the API key in the value field.
```
Comment (sroze) [inline core/jwt.md]: Maybe these 3 sentences can be in the same paragraph?

**Comment by toofff** [comment]
@sroze done

File: core/jwt.md (hunk 7 above)
Comment (toofff) [reply core/jwt.md]: It's better

**Comment by dunglas** [comment]
Thanks @toofff!

**Pull request merged**